README.md

```diff
@@ -74,18 +74,6 @@ yarn test
 yarn test:int
 ```
 
-#### Dev (HTTPS):
-
-Some parts of the site, such as interacting with a Ledger hardware wallet, require an HTTPS environment to work. To develop on HTTPS:
-
-1.  Create your own SSL Certificate (Heroku has a [nice guide here](https://devcenter.heroku.com/articles/ssl-certificate-self))
-2.  Move the `.key` and `.crt` files into `webpack_config/server.*`
-3.  Run the following command:
-
-```bash
-yarn dev:https
-```
-
```

Comment:
I always like to see docs deleted when you don't have to worry about them any more 😉 nice job

Reply:
😎